src/routes.php

```diff
@@ -97,11 +97,11 @@ public function getMember()
         return $this->member;
     }
 
-    public function generateTokenRequestUrl($data, $csrfToken)
+    public function generateTokenRequestUrl($data, $csrfToken, $redirectUrl)
     {
-        $destinationData = json_decode($data['destination'], true);
+        $iban = "DE16700222000072880129";
```

Comment:
shouldn't "iban" be in the query parameters, just so whoever reads the sample knows that it must be provided externally.

Reply:
Mmmm but in the real world it shouldn't be configurable from client side, right? still iban is visible in the server side's source code